source/tutorial/enforce-keyfile-access-control-in-existing-sharded-cluster-no-downtime.txt

```diff
@@ -0,0 +1,297 @@
+==========================================================================
+Enable mandatory user authentication in a sharded cluster without downtime
+==========================================================================
```

Comment:
We tend to use title capitalization for pages.

Also, How is mandatory user authentication different from just user authentication